src/cmr/sizing/formats.clj

```diff
@@ -70,7 +70,7 @@
   (reduce (fn [total-estimate variable]
             (let [avg-gran-size (get-avg-gran-size variable)
                   total-granule-input-bytes (read-string (:total-granule-input-bytes params))
-                  avg-compression-rate (if (= fmt :nc4)
+                  avg-compression-rate (if (= (keyword (string/lower-case fmt)) :nc4)
```

Comment:
Thoughts for later: This might be a good utility function to add to cmr-exchange-common (i.e., `str->kwd`) ...